fix grammar in main readme (#819)
oceanroleplay/discord.ts: packages/discordx/README.md

# ⚔️ [Guards](https://discord-ts.js.org/docs/decorators/general/guard)

> We implemented a guard system that functions like the [Koa](https://koajs.com/) middleware system

Reviewer: ```suggestion
We implemented a guard system that functions same as [Koa](https://koajs.com/) middleware system
```

Reviewer: @VictoriqueMoe

Reviewer: @AndyClausen @VictoriqueMoe

Reviewer 2: "like the" is more correct

Reviewer: ok

Reviewer: okay, thanks
